dist/svg/internal.svg

```diff
@@ -1,79 +1,162 @@
 <?xml version="1.0" encoding="utf-8"?>
 <svg xmlns="http://www.w3.org/2000/svg">
-   <symbol viewBox="0 0 14 14" id="icon-arrow-right-group-titles">
```

Comment:
What's the difference between this internal file and `icons.svg`? They seem have duplication.

Reply:
Gotcha.